CONTRIBUTING.html

```diff
@@ -0,0 +1,1271 @@
+<!DOCTYPE html><html><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1"><style>body {
```

Comment:
Where is this file comming from? :)

Reply:
my local rendering, deleted in latest commit.